### DISCUSSION TITLE
Search bar does not appear at all on iOS 12. Is it not supported?

### DISCUSSION THREAD

**Discussion by SimpleCreations** [software-mansion/react-native-screens#1087] [Q&A] [answered]
Hi!

It seems like the search bar does not appear at all when running the app on iOS 12.
I was wondering if this is intentional and some older versions of iOS are not supported.

I am particularly confused because [this issue (#194)](https://github.com/software-mansion/react-native-screens/issues/194) (after which the search bar support was added) mentions iOS 11, so I assumed the support includes versions all the way back to iOS 11.

**Comment by WoLewicki** [comment, marked as answer]
Can you check if https://github.com/software-mansion/react-native-screens/pull/1115 fixes the issue and does not introduce any new ones?

**Reply by SimpleCreations** [reply]
The fix worked, thank you!
However I've noticed that on iOS 12 the search bar stays on screen during scroll, but on iOS 14 it slides up under the header (this is the expected behavior; I use the default search bar options).
Should I create an issue for this?

**Reply by WoLewicki** [reply]
You can create an issue with a reproduction for sure. Does it work correctly on iOS 13?

**Reply by SimpleCreations** [reply]
I've created two relates issues: #1120 and #1121 
The behavior on iOS 13 matches the (correct) behavior on iOS 14 in both cases
